docs/docs/adrs/adr-008-throttle-retries.md

```diff
@@ -67,46 +89,57 @@ Two things are achieved with this approach:
+A `VSCMaturedPacket` communicates to the provider that sufficient time passed on the consumer since the corresponding `VSCPacket` has been applied (on the consumer) such that infractions committed on the consumer could have been submitted.
 
-If a consumer sends VSCMatured packets too leniently: The consumer is malicious and sending duplicate vsc matured packets, or sending the packets sooner than the ccv protocol specifies. In this scenario, the provider needs to handle vsc matured packets immediately to prevent DOS, state bloat, or other issues. The only possible negative outcome is that the malicious consumer may not be able to jail a validator who should have been jailed. The malicious behavior only creates a negative outcome for the chain that is being malicious.
+If the consumer is following the queuing/blocking protocol described, then no bad behavior occurs and the _VSC Maturity and Slashing Order_ property is maintained.
 
-If a consumer blocks the sending of VSCMatured packets: The consumer is malicious and blocking vsc matured packets that should have been sent. This will block unbonding only up until the VSC timeout period has elapsed. At that time, the consumer is removed. Again the malicious behavior only creates a negative outcome for the chain that is being malicious.
+If a consumer sends `VSCMaturedPackets` too leniently -- the consumer is malicious and sends duplicate `VSCMaturedPackets`, or sends the packets sooner than the CCV protocol specifies -- then the provider needs to handle `VSCMaturedPackets` immediately to prevent DOS, state bloat, or other issues. 
+The only possible negative outcome is that the malicious consumer may not be able to jail a validator who should have been jailed. 
+The malicious behavior only creates a negative outcome for the consumer chain that is being malicious.
+
+If a consumer blocks the sending of `VSCMaturedPackets`, then unbonding operations on the provider will be delayed, but only until the VSC timeout period has elapsed. 
+At that time, the consumer is removed. 
+Again the malicious behavior only creates a negative outcome for the consumer chain that is being malicious.
 
 ### Splitting of PRs and Upgrade Order
 
-This feature will implement consumer changes in [#1024](https://github.com/cosmos/interchain-security/pull/1024). Note these changes should be deployed to prod for all consumers before the provider changes are deployed to prod. That is the consumer changes in #1024 are compatible with the current ("v1") provider implementation of throttling that's running on the Cosmos Hub as of July 2023.
+This feature will implement consumer changes in [#1024](https://github.com/cosmos/interchain-security/pull/1024). 
+
+❗***These changes should be deployed to production for all consumers before the provider changes are deployed to production.*** 
```

Comment:
This is difficult to enforce.

Does this mean that we cannot actually have this on the provider any time soon?

@mpoke

Reply:
We decided to release the throttling v2 changes in two releases: v3.2.0 will contain only the consumer-side changes and it will be backwards compatible with previous versions; v4.0.0 will the provider-side changes and it will not be compatible with consumer running versions < v3.2.0. For more details, see https://github.com/cosmos/interchain-security/pull/1443/commits/2d6b8771af2804314215d2186f2e0b6bb913eb19